.vscode/launch.json

```diff
@@ -4,6 +4,15 @@
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
   "version": "0.2.0",
   "configurations": [
+
+    {
+      "name": "liferay version",
+      "type": "go",
+      "request": "launch",
+      "mode": "debug",
+      "program": "main.go",
+      "args": ["--version"]
+    },
```

Comment:
The list is becoming very long. Would you mind placing the new entry in sorted order?